connexta/eve
# Delete Extra BuildIcon.js

#### What does this PR do?
An extra  `BuildIcon.js` with no content exists in the `src` folder in master. Not sure why this is here, but it should be deleted.

#### Who is reviewing it?

@vinamartin @Kjames5269 @alchucam @mcalcote @haydenudelson

#### How should this be tested?
Make sure removing this file doesn't break the wallboard
```yarn install && yarn build && yarn start```

#### Screenshots

<img width="976" alt="Screen Shot 2019-07-31 at 8 13 24 AM" src="https://user-images.githubusercontent.com/35464088/62224181-1d8a8280-b36b-11e9-92f4-b25f71a727ac.png">

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
